Tests/SwiftlyTests/SwiftlyTests.swift

```diff
@@ -99,25 +99,8 @@ class SwiftlyTests: XCTestCase {
     ]
 
     func baseTestConfig() async throws -> Config {
-        let getEnv = { varName in
-            guard let v = ProcessInfo.processInfo.environment[varName] else {
-                throw SwiftlyTestError(message: "environment variable \(varName) must be set in order to run tests")
-            }
-            return v
-        }
-
-        let name = try? getEnv("SWIFTLY_PLATFORM_NAME")
```

Comment:
were these variables just unused?

Reply:
These variables have been slowly been superseded by platform auto detection, first in the production part of swiftly on macOS, then gradually the tests. This is removing the last remnant of them because I didn't want to miss test coverage of the auto-detection on the different CI platforms since I had modified some of that code here.

@shahmishal can we get the CI to add the new platforms? I think that I added all of the docker files, and docker compose, but the CI didn't pick up Debian 12, Fedora 39, or the two new Ubuntu (24.04, 23.10). I still see the usual RHEL9, Ubuntu 22.04, etc.

Reply:
Switching over to GH actions has yielded these new platforms for testing. See the "checks" section of this PR.